relay-base-schema/src/metrics/mri.rs

```diff
@@ -116,6 +116,10 @@ pub enum MetricNamespace {
     Profiles,
     /// User-defined metrics directly sent by SDKs and applications.
     Custom,
+    /// Metric stats.
+    ///
+    /// Metrics about metrics.
+    MetricStats,
```

Comment:
How about have it only like:
```suggestion
    Stats,
```

Since the we already have `MertricNamespace`, so it should be easy to deduce that this is metrics about metrics. What do you think?

Reply:
It was the plan to name the namespace `metric_stats` in Snuba, since it contains stats of other metrics.

Reply:
I would just rename the enum in Relay, but the string definition could be whatever we already are agreed on.
But I also see your point and I myself do not like when we have such inconsistency which could lead to longer debug sessions later on.